src/default.frag.glsl

```diff
@@ -6,9 +6,9 @@
 
 uniform sampler2DArray textures;
 
-attribute int frag_textureid;
-attribute vec2 frag_texcoord;
+varying vec2 frag_texcoord;
+varying float frag_textureid;
 
 void main() {
-    gl_FragColor = texture(textures, vec3(frag_texcoord, float(frag_textureid)));
-}
\ No newline at end of file
+    gl_FragColor = texture2DArray(textures, vec3(frag_texcoord, float(frag_textureid)));
```

Comment:
Another interesting thing to note. I suppose it makes sense given that sampler2DArrays were considered an extension in OpenGL 2.0.

Reply:
The reason I needed to do this is because on my machine there was no function named `texture`. I assume it's supposed to automatically overload between sampler types. Since I don't have it, I needed to manually specify it